README.md

```diff
@@ -129,3 +125,52 @@ go run examples/generate_balances/main.go \
   validator-data/bootstrap_balances.json
 rosetta-validator check:complete
 ```
+
+### Running on development
+#### Running on RC1:
```

Comment:
should we move RC1 out of development section?

Reply:
why?
It's on this section because it uses `go run main.go` which is what you during development.

Reply:
oh my bad.. I got confused since I confused dev network with development process, since it said `Running on development`. Maybe the title can be changed to `Developing Locally with Celo Rosetta Interface` or something?

Reply:
mmm.. no need imo, it's already within `Developer Guide`